### PR TITLE
Page plan vignette

### PR DESCRIPTION
**Pull request by bzkrouse**

### CODE REVIEW

**Review by kimjj93 — APPROVED** [review]

File: vignettes/page_plan.Rmd [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+  )
+)  %>% 
+  print_to_gt(data_demog2) 
+```
+
+#### Every unique value of 1 variable
+
+Suppose we want to split the table by every unique value of `rowlbl1`. and add a footnote indicating the table grouping. We can drop `rowlbl1` from the tables since its value will be printed in the note. We will use the `gt::grp_pull()` function to print the individual tables nicely in the vignette. 
+```{r  }
+gts <- tfrmt(
```
Comment (statasaurus) [inline vignettes/page_plan.Rmd:115]: is there an easy way to make this shorter and then reference the vignette where it shows how to make this in full. Then you can just focus on the page plan (like we might want to save the pre-made ones as json files in the inst folder and then we can just reference them and say the styling for this tfrmt can be found in this vignette)

**Review by statasaurus — APPROVED** [review]
Looks great there are some tweaks around the edges you can make but it is pretty understandable

File: vignettes/building_blocks.Rmd [hunk 2]
```diff
@@ -29,7 +29,9 @@ Much of the tfrmt is divided into "plans" which provide the blueprint for how th
  
 <p align="center"><img src="../man/figures/table_plans.png" width="900px"></p>
 
-Note: Footnotes also use this systems and have a footnote plan (`footnote_plan`) to define what footnote contents and where they should be applied. 
+Additional features that use this system: 
+- Footnotes can be specified via footnote plan (`footnote_plan`) to define what footnote contents and where they should be applied.
```
[the remark this answers is not in the document]
Comment (bzkrouse) [reply vignettes/building_blocks.Rmd:33]: The footnote just goes on the relevant page.

File: vignettes/page_plan.Rmd (hunk 1 above)
Comment (bzkrouse) [reply vignettes/page_plan.Rmd:115]: Yes! thanks for the suggestion. As discussed, going to make use of `layer_tfrmt` here.

File: vignettes/big_ns.Rmd [hunk 3]
```diff
@@ -159,6 +161,59 @@ with_more_big_n_data <- tribble(
 print_to_gt(n_span_tfrmt, .data = with_more_big_n_data)
 ```
  
+## Page-Level Big Ns
+If our table has a `page_plan` and is to be split by every unique value of a grouping variable, we may desire to have different big Ns for each split of the table. In this case, a user can supply group-level big Ns to the data. In order for this to work properly group levels MUST align to those indicated in the `page_plan` (i.e. any group or label values set to ".default" in the `page_structure`). This functionality can be enabled via the `by_page` argument in `big_n_structure`.
```
[the remark this answers is not in the document]
Comment (bzkrouse) [reply vignettes/big_ns.Rmd]: Resolving this after clean up and discussion.

**Pull request merged**